ext/sqlite3/config0.m4

```diff
@@ -19,46 +19,46 @@ if test $PHP_SQLITE3 != "no"; then
     fi
   fi
 
-  AC_MSG_CHECKING([for sqlite3 files in default path])
-  for i in $PHP_SQLITE3 /usr/local /usr; do
-    if test -r $i/include/sqlite3.h; then
-      SQLITE3_DIR=$i
-      AC_MSG_RESULT(found in $i)
-      break
-    fi
-  done
+    AC_MSG_CHECKING([for sqlite3 files in default path])
```

Comment:
The indentation changes here look wrong.

Reply:
Thanks. Should be resolved now.